tools/rsdl/csdl-xml/lib/xmlSerializer.js

```diff
@@ -0,0 +1,653 @@
+var builder = require('xmlbuilder');
```

Comment:
```suggestion
const builder = require('xmlbuilder');
```

Please use `const` for immutable variables, and `let` for ordinary variables.